docker-compose/api-test.py

```diff
@@ -0,0 +1,79 @@
+import requests
+import unittest
+import random
+import string
+import json
+
+class WalletAPITest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.wallet_url = "http://localhost:7001"
+        cls.headers = {"Content-Type": "application/json" }
+        cls.name = ''.join(random.choices(string.ascii_letters + string.digits, k=8))
+        cls.email = f"{cls.name}@example.com"
+        cls.password = "password"  # Replace with a proper password policy if needed
+        cls.account_type = "email"  # Adjust this based on your actual account types
+        cls.did_method = "jwk"  # Add your own value
+        cls.key_algorithm = "Ed25519"  # Add your own value
+        cls.key_origin = "jwk"  # Add your own value
+        cls.jwk_payload = "{ 'backend': 'jwk', 'keyType': 'Ed25519' }"  # Add your own value
```

Comment:
wouldn't it be better to use json.dumps?
e.g. 
```
json.dumps({"backend":"jwl","keyType":"Ed25519"})
```

because otherwise it throws:
```
Caused by: kotlinx.serialization.json.internal.JsonDecodingException: Unexpected JSON token at offset 2: Encountered an unknown key ''backend'' at path: $
Use 'ignoreUnknownKeys = true' in 'Json {}' builder to ignore unknown keys.
JSON input: { 'backend': 'jwk', 'keyType': 'Ed25519' }
```